plugins-compat-tester/src/main/resources/nonstandardtagplugins.properties

```diff
@@ -0,0 +1,2 @@
+electricflow=cloudbees-flow-%s
+electricflow-minimumVersion=1.1.8
```

Comment:
Is not more interesting to have the maximunVersion? If a plugin changes, it is to become compliant with the rules, so the latest versions most likely, don't need the hook anymore. Or maybe both properties. WDYT? 

BTW: I was working in exactly the same hook but specifically in a job on our CI, so :100: